site/_posts/2016-12-05-last-week-in-manageiq-make-hawkular-tenant-changeable.html.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Last Week in ManageIQ: Last Week in ManageIQ: Make Hawkular tenant changeable"
+author: jvlcek
+date: 2016-12-02 11:50:57 -05:00
```

Comment:
Please change the date to Dec 5 or 6 (i.e. this week, as you're writing about "last week"). Thanks!